### DISCUSSION TITLE
Is turbo pack available for the public?

### DISCUSSION THREAD

**Discussion by emadbaqeri** [vercel/turborepo#3360] [Help] [answered]
I'm wondering if the `turbopack` is available for production. I have switched the monorepo manager from `nx` to `turborepo` and just saw how excellent and handy this tool is. Let me know if we can configure our project bundling feature to use `turbopack`. Is not let me know how can I help to bring in to the community use stage as soon as possible ;)

**Comment by mehulkar** [comment, marked as answer]
Turbopack is currently available as an opt-in feature for Next.js's dev server: You can see roadmap updates at https://turbo.build/pack/docs/roadmap and look at [Open issues](https://github.com/vercel/turbo/issues?q=is%3Aopen+is%3Aissue+label%3A%22area%3A+turbopack%22) to contribute!
